lib/msal-core/src/utils/TimeUtils.ts

```diff
@@ -26,4 +26,11 @@ export class TimeUtils {
     static now(): number {
         return Math.round(new Date().getTime() / 1000.0);
     }
+
+    /**
+     * Returns the amount of time in milliseconds since the page loaded.
+     */
+    static relativeNowMs(): number {
+        return window.performance.now();
```

Comment:
`window.performance.now` returns high resolution duration in milliseconds, relative to the start of the page load. We don't need to use `Date`, since we are calculating a relative duration. https://developer.mozilla.org/en-US/docs/Web/API/Performance/now#Example